uber/templates/regform.html

```diff
@@ -2,7 +2,7 @@
 
 {#- Setting some variables to keep the size of conditional clauses smaller. -#}
 {%- set is_prereg_form = c.PAGE_PATH in ['/preregistration/form', '/preregistration/post_form'] -%}
-{%- set is_prereg_dealer = c.PAGE_PATH in ['/preregistration/dealer_registration'] -%}
+{%- set is_prereg_dealer = c.PAGE_PATH in ['/preregistration/dealer_registration'] or attendee.badge_type == c.PSEUDO_DEALER_BADGE -%}
```

Comment:
I found this while testing -- if there's a validation error, the URL changes, causing `is_prereg_dealer` to erroneously read as false.